controllers/nginx/pkg/config/config.go

```diff
@@ -301,6 +304,7 @@ func NewDefault() Configuration {
 		SSLSessionTimeout:        sslSessionTimeout,
 		UseGzip:                  true,
 		WorkerProcesses:          strconv.Itoa(runtime.NumCPU()),
+		LoadBalanceAlgorithm:     "least_conn;",
```

Comment:
don't leave `;` as part of the setting. That character must be present in the template instead